lib/web/apiserver.go

```diff
@@ -1063,6 +1063,7 @@ func (h *Handler) bindDefaultEndpoints() {
 
 	// AWS IAM Roles Anywhere Integration Actions
 	h.GET("/webapi/scripts/integrations/configure/awsra-trust-anchor.sh", h.WithLimiter(h.awsRolesAnywhereConfigureTrustAnchor))
+	h.POST("/webapi/sites/:site/integrations/aws-ra/:name/listprofiles", h.WithClusterAuth(h.awsRolesAnywhereListProfiles))
```

Comment:
i think this could be a `h.Get` and have `nextToken` be part of the URL query param like [list unified resources](https://github.com/gravitational/teleport/blob/dbb20a70447a906d6d743a0060a47e33e95b15d6/lib/web/apiserver.go#L844) (we also seem to consistently call it `startKey` in the URL query param)

Reply:
I think I have to get back to a `POST`
Our router panics if I set this to a `GET` because of conflicting routes:
```
panic: 'aws-ra' in new path '/webapi/sites/:site/integrations/aws-ra/:name2/listprofiles' conflicts with existing wildcard ':name' in existing prefix '/webapi/sites/:site/integrations/:name'
```

Reply:
Personally I don't love the idea of changing the method as a workaround to our URL format, but not a hill I'm willing to die on.